wasm/package.json

```diff
@@ -35,6 +35,8 @@
   "devDependencies": {
     "@types/chai": "^4.3.0",
     "@types/mocha": "^9.1.0",
+    "@types/node": "^18.7.18",
+    "@types/webextension-polyfill": "^0.9.0",
```

Comment:
Theoretically the webextension storage adapter should be part of the browser extension, but this is not a big dependency, can live within wallet-core/wasm as well.

Reply:
This is dev dependency for testing `Storage` types